### PR TITLE
e2e token tests

### PR DESCRIPTION
**Pull request by saiakilesh**
according to #101 #102 #103

### CODE REVIEW

File: dkg-test-suite/tests/tokenAddProposal.test.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+describe('Token Add Proposal', () => {
+	test('should be able to sign token add proposal', async () => {
+        const anchor = signatureBridge.getAnchor(localChain.chainId, ethers.utils.parseEther('1'))!;
+        const governedTokenAddress = anchor.token!;
+		let governedToken = GovernedTokenWrapper.connect(governedTokenAddress , wallet1);
+        const resourceId = await governedToken.createResourceId();
+        // Create Mintable Token to add to GovernedTokenWrapper
+           //Create an ERC20 Token
+        const tokenToAdd = await MintableToken.createToken('testToken', 'TEST', wallet1);
```
Comment (shekohex) [inline dkg-test-suite/tests/tokenAddProposal.test.ts:47]: Spaces & tabs, format your files, please.

File: dkg-test-suite/tests/tokenRemoveProposal.test.ts [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+	polkadotApi,
+	signatureBridge,
+	wallet1,
+	wallet2,
+	charlieNode,
+	localChain2, executeAfter
+} from './utils/util';
+
+
+describe('Token Add Proposal', () => {
```
Comment (shekohex) [inline dkg-test-suite/tests/tokenRemoveProposal.test.ts:44]:
```suggestion
describe('Token Remove Proposal', () => {
```

File: dkg-test-suite/tests/tokenRemoveProposal.test.ts [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+	test('should be able to sign token add proposal', async () => {
```
Comment (shekohex) [inline dkg-test-suite/tests/tokenRemoveProposal.test.ts:45]:
```suggestion
	test('should be able to sign token remove proposal', async () => {
```

File: dkg-test-suite/tests/wrappingFeeUpdateProposal.test.ts [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+	signatureBridge,
+	wallet1,
+	wallet2,
+	charlieNode,
+	localChain2, executeAfter
+} from './utils/util';
+
+describe('Wrapping Fee Update Proposal', () => {
+
+	test('should be able to sign wrapping fee update proposal', async () => {
```
Comment (shekohex) [inline dkg-test-suite/tests/wrappingFeeUpdateProposal.test.ts:44]: Do another negative test. Where the wrapping fee is over 100. and it should fail ofc.

**Review by shekohex — CHANGES_REQUESTED** [review]
Looks good so far! just a few changes

File: dkg-test-suite/tests/tokenRemoveProposal.test.ts [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+				signed: {
+					kind: 'TokenAdd';
+					data: HexString;
+					signature: HexString;
+				};
+			};
+			// sanity check.
+			expect(dkgProposal.signed.data).toEqual(prop);
+			// perfect! now we need to send it to the signature bridge.
+			// but first, we need to log few things to help us to debug.'
```
Comment (dutterbutter) [inline dkg-test-suite/tests/tokenRemoveProposal.test.ts:107]: I don't think these comments are needed.

File: dkg-test-suite/tests/tokenRemoveProposal.test.ts [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+			const tx2 = await contract.executeProposalWithSignature(
+				dkgProposal.signed.data,
+				dkgProposal.signed.signature
+			);
+			await expect(tx2.wait()).toResolve();
+			// Want to check that token was actually added
+			expect((await governedToken.contract.getTokens()).includes(tokenToAdd.contract.address)).toBeTrue();
+		}
+		await sleep(5 * BLOCK_TIME);
+		//const tokenToRemove = await MintableToken.createToken('testToken', 'TEST', wallet1);
```
Comment (dutterbutter) [inline dkg-test-suite/tests/tokenRemoveProposal.test.ts:125]: Remove commented out line please.

File: dkg-test-suite/tests/tokenRemoveProposal.test.ts [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+			signed: {
+				kind: 'TokenRemove';
+				data: HexString;
+				signature: HexString;
+			};
+		};
+		// sanity check.
+		expect(dkgProposal.signed.data).toEqual(prop);
+		// perfect! now we need to send it to the signature bridge.
+		// but first, we need to log few things to help us to debug.'
```
Comment (dutterbutter) [inline dkg-test-suite/tests/tokenRemoveProposal.test.ts:179]: Remove unneeded comments please.

File: dkg-test-suite/tests/wrappingFeeUpdateProposal.test.ts [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+			signed: {
+				kind: 'WrappingFeeUpdate';
+				data: HexString;
+				signature: HexString;
+			};
+		};
+		// sanity check.
+		expect(dkgProposal.signed.data).toEqual(prop);
+		// perfect! now we need to send it to the signature bridge.
+		// but first, we need to log few things to help us to debug.'
```
Comment (dutterbutter) [inline dkg-test-suite/tests/wrappingFeeUpdateProposal.test.ts:104]: Remove unneeded comments please.

**Review by dutterbutter — CHANGES_REQUESTED** [review]

**Review by shekohex — APPROVED** [review]

**Pull request merged**